mssql-cli.sh

```diff
@@ -1,7 +1,9 @@
 #! /bin/bash -e
 
-docker run $* \
```

Comment:
You have changed the behavior of this script.
I can no longer accept -it in order to launch an interactive CLI.

Reply:
They need to be runnable by humans, e.g. with -it.In many ways, the ideal default behavior would be docker run -it unless its overridden.Also, I think that instead of supplying a script on the command line you can simple do db-mysql.sh < script.sql